test/internal/laws/Chain.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var chain = require('./chain_');
+var compose = require('./compose');
+var forall = require('./forall');
+
+
+module.exports = function(equals) {
+  return forall({
+
+    //  m >>= f >>= g = m >>= (f >=> g)
+    associativity: function(m, f, g) {
+      var lhs = chain(g)(chain(f)(m));
+      var rhs = compose(chain(g))(chain(f))(m);
```

Comment:
I think this law hasn't been updated.

if we use kleisli compose than we need 3 functions instead of 2 (f, g)
```js
var lhs = composeKleisli(f, composeKleisli(g, h))(m);
var rhs = composeKleisli(composeKleisli(f, g), h))(m);
```